src/components/header/Header.jsx

```diff
@@ -0,0 +1,160 @@
+import { useEffect, useState } from "react";
+
+const Header = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [activeLink, setActiveLink] = useState("WORK");
+
+  const toggleMenu = () => {
+    setIsMenuOpen(!isMenuOpen);
+  };
+
+  const setActive = (link) => {
+    setActiveLink(link);
+  };
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerWidth > 768) {
+        setIsMenuOpen(false);
+      }
+    };
+
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
+  return (
+    <div className="fixed top-0 w-full bg-white z-50  ">
+      <div className=" container mx-auto px-3">
+        <div className="flex justify-between  items-center py-4">
+          <div className="flex  gap-2 md:gap-8  justify-center items-center">
+            <img src="/src/assets/logo.svg" alt="" className="" />
+            <img src="/src/assets/logo+.svg" alt="" />
+          </div>
+          <div className="hidden md:block">
+            <ul className="flex gap-24 md:gap-5 xl:gap-24  lg:gap-16  items-center font-semibold text-gray-400 ">
+              <li>
+                <a
+                  href="#"
+                  className={`text-gray-400 ${
+                    activeLink === "WORK" ? "text-gray-700" : ""
+                  }`}
+                  onClick={() => setActive("WORK")}
```

Comment:
Please do not use the arrow function in the template. instead, create one function and use the same.